src/block_service/hdfs/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(MY_PROJ_NAME dsn.block_service.hdfs)
+
+#Source files under CURRENT project directory will be automatically included.
+#You can manually set MY_PROJ_SRC to include source files under other directories.
+set(MY_PROJ_SRC "")
+
+#Search mode for source files under CURRENT project directory ?
+#"GLOB_RECURSE" for recursive search
+#"GLOB" for non - recursive search
+set(MY_SRC_SEARCH_MODE "GLOB")
+
+set(MY_PROJ_LIBS "")
+
+#Extra files that will be installed
+set(MY_BINPLACES "")
+
+dsn_add_static_library()
+
+target_link_libraries(${MY_PROJ_NAME} PUBLIC hdfs)
```

Comment:
`target_link_libraries(${MY_PROJ_NAME} PUBLIC hdfs ${JAVA_JVM_LIBRARY})` is required, because libraries(hdfs,jvm) are for both a target and its dependents.